codegenerator/cli/templates/static/codegen/src/InMemoryTable.res

```diff
@@ -157,17 +157,21 @@ module Entity = {
     inMemTable: t<'entity>,
     entityUpdate: Types.entityUpdate<'entity>,
     ~shouldSaveHistory,
+    ~containsRollbackDiffChange=false,
   ) => {
     //New entity row with only the latest update
-    let newEntityRow = Types.Updated({
+    @inline
+    let newEntityRow = () => Types.Updated({
       latest: entityUpdate,
       history: shouldSaveHistory ? [entityUpdate] : [],
+      // For new entities, apply "containsRollbackDiffChange" from param
+      containsRollbackDiffChange,
```

Comment:
We now keep track in the in memory store if the entity update contains changes from a rollback diff